libtransmission/transmission.h

```diff
@@ -1587,7 +1587,8 @@ struct tr_info
     tr_piece_index_t pieceCount;
 
     /* General info */
-    uint8_t hash[SHA_DIGEST_LENGTH];
+    tr_sha1_digest_t hash;
+    // uint8_t hash[SHA_DIGEST_LENGTH];
```

Comment:
Remove the comment.

Reply:
fixed 997610d191c6ac6b23be4151af97408efc4ddb34